py/src/braintrust/cli/install/api.py

```diff
@@ -426,24 +429,24 @@ def main(args):
             if len(org_info) == 1:
                 org_info = org_info[0]
 
-        if org_info and (
-            (function_url and org_info["api_url"] != function_url)
-            or (proxy_url and org_info["proxy_url"] != proxy_url)
-        ):
-            _logger.info(f"Will update org {org_info['name']}")
+        if org_info and (universal_url and org_info["api_url"] != universal_url):
+            _logger.info(f"Will update org {org_info['name']}'s urls.")
             _logger.info(f"  They are currently set to:")
             _logger.info(f"  API URL: {org_info['api_url']}")
             _logger.info(f"  Proxy URL: {org_info['proxy_url']}")
             _logger.info(f"And will update them to:")
 
             patch_args = {"id": org_info["id"]}
-            if function_url and org_info["api_url"] != function_url:
-                patch_args["api_url"] = function_url
-                _logger.info(f"  API URL: {function_url}")
-            if proxy_url and org_info["proxy_url"] != proxy_url:
-                patch_args["proxy_url"] = proxy_url
-                _logger.info(f"  Proxy URL: {proxy_url}")
+            if universal_url and org_info["api_url"] != universal_url:
+                patch_args["api_url"] = universal_url
+                patch_args["is_universal_api"] = True
+                _logger.info(f"  API URL: {universal_url}")
+                _logger.warn(
+                    f"\nNOTE: You can delete the proxy URL from your org settings now. It is no longer needed."
+                )
+
+            # Make the actual request
             app_conn().post(
                 "api/organization/patch_id",
                 json=patch_args,
-            )
+            ).raise_for_status()
```

Comment:
fwiw `response_raise_for_status` in `util.py` generally provides a better error message, since it prints out the `resp.text` in addition to `resp.status_text`.